### PR TITLE
Minor doc fixes/improvements

### PR DESCRIPTION
**Pull request by oliver**
### What is the purpose of this change? What does it change?

Two minor doc fixes, one minor doc improvement.

### Was the change discussed in an issue or in the forum before?

No, not that I know of.

### Checklist

- [x] I have read the [Contribution Guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches)
- [ ] I have added tests for all changes in this PR
- [ ] I have added documentation for the changes (in the manual)
- [ ] There's a new file in `changelog/unreleased/` that describes the changes for our users (template [here](https://github.com/restic/restic/blob/master/changelog/changelog-entry.tmpl))
- [ ] I have run `gofmt` on the code in all commits
- [x] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits)
- [x] I'm done, this Pull Request is ready for review

### CODE REVIEW

File: doc/040_backup.rst [hunk 1]
```diff
@@ -93,7 +93,7 @@ Basic example:
     *.go
     # exclude foo/x/y/z/bar foo/x/bar foo/bar
     foo/**/bar
-    $ restic -r /tmp/backup backup ~/work --exclude=*.c --exclude-file=exclude
+    $ restic -r /tmp/backup backup ~/work --exclude="*.c" --exclude-file=exclude
```
Comment (fd0) [inline doc/040_backup.rst]: For the record: This is not strictly necessary, but I think it's a good idea. Shells will resolve `--exclude *.c`, but won't resolve `--exclude=*.c`.

File: doc/040_backup.rst [hunk 2]
```diff
@@ -126,7 +126,7 @@ backup from a file. This is especially useful if a lot of files have to
 be backed up that are not in the same folder or are maybe pre-filtered
 by other software.
 
-or example maybe you want to backup files that have a certain filename
+For example maybe you want to backup files that have a certain filename
```
Comment (fd0) [inline doc/040_backup.rst]: Good catch!

File: doc/040_backup.rst [hunk 3]
```diff
@@ -146,6 +146,9 @@ args:
 
     $ restic -r /tmp/backup backup --files-from /tmp/files_to_backup /tmp/some_additional_file
 
+Pathes in the listing file should be absolute. Relative pathes will be
```
Comment (fd0) [inline doc/040_backup.rst]: The plural of "path" is "paths", can you please correct that?

I'm not sure if it's a good idea to recommend absolute paths for a file used with `--files-from`. There are plenty of use cases where it's important to specify relative paths. I'm fine with the hint that relative paths will be interpreted in context with the current directory (although that's basic knowledge the reader needs to be familiar with anyway), but I think we should not recommend that the file names in the file should be absolute. Can you remove the first sentence here? Thanks!

**Review by fd0 — CHANGES_REQUESTED** [review]

**Comment by fd0** [comment]
Thanks for taking the time to improve the documentation, it means a lot for the project! :)

**Comment by oliver** [comment]
Oops, I had made a mistake when experimenting with that option, and had thought that the paths are interpreted as relative to the listing file location (not relative the CWD).

You're right, it is not necessary to document this behaviour for relative paths since they behave just as expected. I would add a note that both kinds of paths are allowed; but maybe even that note is superfluous.

**Comment by fd0** [comment]
> I had made a mistake when experimenting with that option, and had thought that the paths are interpreted as relative to the listing file location (not relative the CWD).

Ah, that explains the desire to document this ;)

**Review by fd0 — APPROVED** [review]

**Pull request merged**